src/route_progress_cost.cpp

```diff
@@ -56,8 +56,11 @@ float RouteProgressCost::Evaluate(Time t, const VectorXf& input) const {
 
   const float desired_route_pos =
       initial_route_pos_ + (t - initial_time_) * nominal_speed_;
-
-  const Point2 desired = polyline_.PointAt(desired_route_pos);
+  bool is_endpoint;
+  const Point2 desired = polyline_.PointAt(desired_route_pos, nullptr, nullptr, &is_endpoint);
+  if(is_endpoint){// endpoint fix
```

Comment:
see earlier comments about whitespace and 0.0

Reply:
done!